Parsers/OracleDatabaseAudit/OracleDatabaseAuditEvent.txt

```diff
@@ -0,0 +1,230 @@
+// Usage Instruction :
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias as OracleDatabaseAuditEvent.
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. OracleDatabaseAuditEvent | take 10).
+// Reference : Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+let oracledb_type_1 =() {
+Syslog
+| where SyslogMessage matches regex @".*ACTION(\s)?:\[\d+\]\s\'(.*?)\s"
```

Comment:
There is no SyslogMessage in your sample JSON.  It look like the JSON provided is already parsed fields?

Reply:
Yes, provided sample JSON is already parsed.